requirements/base.txt

```diff
@@ -1,16 +1,16 @@
 #
-# This file is autogenerated by pip-compile with python 3.10
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.8
```

Comment:
Why was it changed? Why are we using 3.8 instead of 3.10?

Reply:
That's my bad @Alec4r, by default i use python3.8, i just updated it again with python.310, thanks